tests/app/data_model/test_questionnaire_store.py

```diff
@@ -5,6 +5,7 @@
 from app.data_models.answer_store import AnswerStore
```

Comment:
We should have a test to show that it always uses the metadata value, so state value changes if metadata value changes.

Reply:
I think this isn't needed now, Lewis rightly pointed out the code I added for QuestionnaireStore wasn't being used so I just removed the attribute